README.md

```diff
@@ -57,6 +57,7 @@ we move them into the [Alumni section](#alumni-) below.
 | Explainer | Issues | Feedback | Group |
 | --------- | ------ | -------- | -------- |
 | [CSS Gap Decorations](CSSGapDecorations/explainer.md) | <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/labels/CSSGapDecorations">![GitHub issues by-label](https://img.shields.io/github/issues/MicrosoftEdge/MSEdgeExplainers/CSSGapDecorations?label=issues)</a> | [New issue...](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/new?assignees=kbabbitt&labels=CSSGapDecorations&template=css-gap-decorations.md&title=%5BCSS+Gap+Decorations%5D+%3CTITLE+HERE%3E) | CSS |
+| [CSS ::tooltip Pseudo Element](CSSTooltipPseudo/explainer.md) | <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/labels/CSSTooltipPseudo">![GitHub issues by-label](https://img.shields.io/github/issues/MicrosoftEdge/MSEdgeExplainers/CSSTooltipPseudo?label=issues)</a> | [New issue...](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/new?template=tooltip-pseudo.md) | CSS |
```

Comment:
Can we move to the brainstorming section?

Reply:
Since the CSSWG resolved to standardize this https://github.com/w3c/csswg-drafts/issues/8930#issuecomment-1644333441 would this still be considered brainstorming? There are open questions, but I don't know if that means it shouldn't be considered active. Not sure what the bar is though?